### PR TITLE
Remove api reference link on webpay modal

### PR DESCRIPTION
**Pull request by gonzunigad**
Remove api reference link on webpay modal

### CODE REVIEW

File: documentacion/webpay-modal/README.md [hunk 1]
```diff
@@ -65,7 +57,7 @@ para identificar tu transacción.
 <div class="language-simple" data-multiple-language></div>
 
 ```php
-response.getToken();
+$response.getToken();
```
Comment (ffiebig) [inline documentacion/webpay-modal/README.md:60]: ->  ?

**Review by ffiebig — CHANGES_REQUESTED** [review]

File: documentacion/webpay-modal/README.md (hunk 1 above)
Comment (gonzunigad) [reply documentacion/webpay-modal/README.md:60]: In php variables should start with $.

Comment (gonzunigad) [reply documentacion/webpay-modal/README.md:60]:
```suggestion
$response->getToken();
```

**Comment by gonzunigad** [comment]
@ffiebig done

**Pull request merged**